portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomRecommendation() {
   const recommendationContainer = document.getElementById('recommendation-container');
   recommendationContainer.innerText = recommendation;
 }
+
+function getMessage() {
+    fetch('/messages')
+    .then(response => response.json())
+    .then((message) => {
+      document.getElementById('message-container').innerText = message[0];
```

Comment:
This is always returning the first message in the array, is this getMessage function intend to do this? If intended, consider renaming the function to something like getFirstMessage.